setup.py

```diff
@@ -19,9 +20,29 @@ def get_console_scripts():
             basename, ext = os.path.splitext(os.path.basename(filename))
             if basename in ['__init__', 'dochelpers'] or ext != '.py':
                 continue
-            yield '{} = qubesadmin.tools.{}:main'.format(
-                basename.replace('_', '-'), basename)
+            yield basename.replace('_', '-'), 'qubesadmin.tools.{}'.format(basename)
 
+# create simple scripts that run much faster than "console entry points"
+class CustomInstall(setuptools.command.install.install):
+    def run(self):
+        bin = os.path.join(self.root, "usr/bin")
+        try:
+            os.makedirs(bin)
+        except:
+            pass
+        for file, pkg in get_console_scripts():
+           path = os.path.join(bin, file)
+           with open(path, "w") as f:
+               f.write(
+"""#!/usr/bin/python3
+from {} import main
+import sys
+if __name__ == '__main__':
+	sys.exit(main())
+""".format(pkg))
+
+           os.chmod(path, 0o755)
+        setuptools.command.install.install.run(self)
```

Comment:
core-admin could use this optimization too. It would be specifically good for `qrexec-policy` tool.